geos/meta.yaml

```diff
@@ -8,8 +8,7 @@ source:
     - cmake.win.patch  # [win]
 
 build:
-  number: 3  # [win]
-  number: 2  # [not win]
+  number: 3
```

Comment:
Maybe the slower `AppVeyor` might choke, but Circle and Travis finished building `geos` just fine.

PS:  Do you want separate build numbers for Windows and *nix?